src/Apps/Order2/Routes/Checkout/Components/Order2FulfillmentDetailsStep/Order2FulfillmentDetailsCompletedView.tsx

```diff
@@ -0,0 +1,56 @@
+import CheckmarkIcon from "@artsy/icons/CheckmarkIcon"
+import { Box, Clickable, Flex, Text } from "@artsy/palette"
+
+interface Order2FulfillmentDetailsCompletedViewProps {
+  fulfillmentDetails: any
+  fulfillmentOption: any
+  onClickEdit: () => void
+}
+export const Order2FulfillmentDetailsCompletedView = ({
```

Comment:
Seeing the [PR](https://github.com/artsy/force/pull/15600) on the payment section, let's standardize the naming and the organization!

Reply:
Yes, agreed.